src/v3/src/components/ReminderPrompt/ReminderPrompt.test.tsx

```diff
@@ -58,9 +59,10 @@ describe('ReminderPrompt', () => {
     const { container, getByRole } = render(<ReminderPrompt {...props} />);
 
     expect(container.firstChild).toBeNull();
-
     act(() => {
-      jest.advanceTimersByTime(DEFAULT_TIMEOUT_MS);
+      // @ts-expect-error setSystemTime does not exist on type 'typeof jest'
+      jest.setSystemTime(Date.now() + DEFAULT_TIMEOUT_MS);
```

Comment:
ts does not recognize setSystemTime as a valid function

Reply:
I think we need to update `@types/jest` version to match the jest version